src/GovUk.Education.ExploreEducationStatistics.Content.Model/Repository/MethodologyVersionRepository.cs

```diff
@@ -87,74 +88,39 @@ await _contentDbContext.Entry(methodology)
             return await _contentDbContext
                 .MethodologyVersions
                 .Where(mv =>
-                    mv.Methodology.Slug == slug
-                    && mv.Methodology.LatestPublishedVersionId == mv.Id)
+                    mv.Methodology.LatestPublishedVersionId == mv.Id
```

Comment:
The alternative, which we've used before for getting around restrictions with JSON columns is to do something like:

```
    return (await _contentDbContext
        .MethodologyVersions
        .Where(mv => mv.Methodology.LatestPublishedVersionId == mv.Id)
        // Pull into memory
        .ToListAsync())
        .SingleOrDefault(mv => slug == mv.Slug);
```

Not sure if there's an argument for one or other approach here tbh - if mv.Slug had a more complex logic then perhaps a stronger argument for this method, but hey!

Reply:
I guess this alternative isn't going to be as efficient, but then if someone changes `Slug` in the future it'll be clear that this also needs to be changed.

I think I'll leave it - if we pull it into memory we need the DB, it seems a bit wasteful to return every `LatestPublishedVersion` including all fields for those methodology versions.